api/tests/Feature/WeatherControllerTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Bus;
+use Tests\TestCase;
+
+class WeatherControllerTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function test_update_weather()
+    {
+        Bus::fake();
+
+        $user1 = User::factory()->create(['latitude' => 12.34, 'longitude' => 56.78]);
+        $user2 = User::factory()->create(['latitude' => 23.45, 'longitude' => 67.89]);
+
+        $this->mock(User::class, function ($mock) use ($user1, $user2) {
```

Comment:
This mock feels unnecessary? You're creating the users in the db so I think it's appropriate to make sure they are pulled out by Eloquent rather than mocking it the whole model.